CMakeLists.txt

```diff
@@ -635,7 +636,7 @@ file(WRITE "${CMAKE_BINARY_DIR}/OpenColorIOConfig.cmake"
     
     ## targets libraries + associated definitions
     if(NOT TARGET OpenColorIO)
-        include(\"\${OpenColorIO_DIR}/cmake/OpenColorIO.cmake\") ## thanks to imported target
+        include(\"\${OpenColorIO_DIR}/${CMAKE_INSTALL_LIBDIR}/cmake/OpenColorIO/OpenColorIO.cmake\") ## thanks to imported target
```

Comment:
Since I am modifying only the path of the file, I'll link to the original CMakeFiles.txt file. I had to trace what this was doing to answer your question. I think this include statement is trying to include the file made by this line of the config: https://github.com/imageworks/OpenColorIO/blob/master/CMakeLists.txt#L626

This code is in a block of text that is being saved to a new file called OpenColorIOConfig.cmake starting at the next line: https://github.com/imageworks/OpenColorIO/blob/master/CMakeLists.txt#L627

I think this is what's going on. The files installed on my system are:
/usr/lib64/cmake/OpenColorIO/OpenColorIO.cmake
/usr/lib64/cmake/OpenColorIO/OpenColorIOConfig.cmake

Hope this helps. Hope I am right in this analysis. :D

Reply:
Thanks for looking into that. It seems this bit of code is a bit gnarly and is importing the installed `OpenColorIO.cmake`...

We will likely try to clean this entire section up later... cough cough @Shootfast

Reply:
Instead of generating the file in the CMakeFiles.txt file, you could make it an external file with variables, much like the .pc files. Then you this would make CMakeFiles.txt a bit cleaner. But I agree, something for the future. It's best to work in small steps than everything at once. :)